docs/examples-profiling.md

```diff
@@ -6,8 +6,25 @@ well your Wasm module is performing! From time to time you might want to dive a
 bit deeper into the performance of your Wasm, and this is where profiling comes
 into the picture.
 
-Profiling support in Wasmtime is still under development, but if you're using
-either [perf](./examples-profiling-perf.md) or
-[VTune](./examples-profiling-vtune.md) the examples in these sections are
-targeted at helping you get some information about the performance of your Wasm
-modules.
+For best results, ideally you'd use hardware performance counters for your
+timing measurements. However, that requires special support from your CPU and
+operating system. Because Wasmtime is a JIT, that also requires hooks from
+Wasmtime to your platform's native profiling tools.
+
+As a result, Wasmtime support for native profiling is limited to certain
+platforms. See the following sections of this book if you're using these
+platforms:
+
+- On Linux, we support [perf](./examples-profiling-perf.md).
+- For Intel's x86 CPUs on Linux or Windows, we support
+  [VTune](./examples-profiling-vtune.md).
```

Comment:
Mind rewording the seconds around here so the list here can be a 3-bullet list with perf/vtune/guest profiling? I suspect many will scan over this page, see two bullets, and deduce there's two means of profiling Wasmtime. I'm sure many more yet will read further and realize there's a third, but having them all in one organized bulleted list I think makes sense (with all the existing commentary about which is appropriate of course)

Reply:
I think it's useful to have the distinction and separation between native and guest profilers, but I've added a bullet point to direct people to keep reading; do you think that takes care of it?